bigframes/dataframe.py

```diff
@@ -2316,11 +2317,20 @@ def melt(
     _NON_NUMERICAL_DESCRIBE_AGGS = ("count", "nunique")
 
     def describe(self, include: None | Literal["all"] = None) -> DataFrame:
+
+        allowed_non_numerical_types = {
+            bigframes.dtypes.STRING_DTYPE,
+            bigframes.dtypes.BOOL_DTYPE,
+            bigframes.dtypes.BYTES_DTYPE,
+        }
+
         if include is None:
             numeric_df = self._drop_non_numeric(permissive=False)
             if len(numeric_df.columns) == 0:
                 # Describe eligible non-numerical columns
-                result = self._drop_non_string().agg(self._NON_NUMERICAL_DESCRIBE_AGGS)
+                result = self._filter_columns_by_type(allowed_non_numerical_types).agg(
+                    self._NON_NUMERICAL_DESCRIBE_AGGS
+                )
             else:
                 # Otherwise, only describe numerical columns
                 result = numeric_df.agg(self._NUMERICAL_DISCRIBE_AGGS)
```

Comment:
Might be worth fixing this typo in this PR `DISCRIBE` -> `DESCRIBE`

Reply:
Good catch